examples/k8s.yaml

```diff
@@ -44,11 +52,11 @@ spec:
             cpu: 100m
             memory: 100Mi
         ports:
-        - containerPort: 55678
-        - containerPort: 55679
-#        - containerPort: 14267
-#        - containerPort: 14268
-#        - containerPort: 9411
+        - containerPort: 55678 # Default Opencensus receiver port.
```

Comment:
In the current ConfigMap for otel-agnet, the open census endpoint is not set

https://github.com/open-telemetry/opentelemetry-collector/blob/be399dcef5cbc4d65c094140b34c12fcfd017b6f/examples/k8s.yaml#L12

I am not sure if it means the open census receiver is open or not, while the container port 55678 is open here.

Reply:
OpenCensus receiver has a default port so once it is specified in the receiver list it will be accepting traffic on 55678.

Reply:
Thanks for the explanation.